plugins/@grouparoo/mailchimp/src/initializers/plugin.ts

```diff
@@ -34,8 +34,16 @@ export class Plugins extends Initializer {
         MailchimpAppTemplate,
         MailchimpSourceTemplate,
         MailchimpPropertyTemplate,
-        MailchimpEmailDestinationTemplate,
-        MailchimpIdDestinationTemplate,
+        new DestinationTemplate(
+          "mailchimp:email",
+          [path.join(templateRoot, "destination", "email", "*.template")],
+          ["sync", "additive", "enrich"]
+        ),
```

Comment:
We can now pass another argument to the `DestinationTemplate` constructor that specifies the default sync mode. You'll probably need to rebase on `feat/sync-modes` to be able to use it.

https://github.com/grouparoo/grouparoo/blob/feat/sync-modes/plugins/%40grouparoo/pipedrive/src/initializers/plugin.ts#L40